docs/easy-deployments.md

```diff
@@ -0,0 +1,37 @@
+# Easy Deployments
+
+Before you follow any of these instructions make sure you first [fork this repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and follow the instructions in [docs/build-from-source.md](docs/build-from-source.md).
+
+There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.
+
+Currently supported platforms:
+
+- [Github Pages](#github-pages). It's a free web hosting service offered by Github. The main Github Pages advantages are that it's free of charge and easy to use.
+ 
+## Github Pages
+
+You can learn more about github pages [here](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages). 
+
+### Technical limitations
+
+Github pages is provided free of charge but it comes with some [usage limits](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits).
+In very general terms, this means that a maximum of about 20.000 **NEW** users can visit your app each month (1). If you grow faster than that, you should probably consider investing more in your hosting platform.
+
+(1) Based on 100GB Github Pages bandwidth / 5MB ott-web-app build dir size and assuming perfect browser cache by all visitors.
+
+### Usage instructions
+
+1. Make sure you can [build this project from source](./build-from-source.md). You need to install optional dependencies!
+1. Make sure you enabled Github Pages feature for `gh-pages` branch in your repository (see [Github Pages Getting Started docs](https://docs.github.com/en/pages/getting-started-with-github-pages))).
+1. Run `yarn deploy:github` and follow instructions on screen.
+
+If you need some customization option check out `yarn deploy:github --help`.
+
+### Technical documentation
+
+`yarn deploy:github` executes a simple nodejs script located in `scripts/deploy-github.js`. That script does following things in this order:
+
+- Runs `yarn build` with `SNOWPACK_PUBLIC_BASE_URL` envvar. That envvar is used to set the url location of the project. By default if SNOWPACK_PUBLIC_BASE_URL is empty the value will be based on the `git remote get-url origin` command. You can also pass your own SNOWPACK_PUBLIC_BASE_URL envvar by running `SNOWPACK_PUBLIC_BASE_URL=/my-base/ yarn deploy:github`.
+- Runs `yarn gh-pages -o origin -d build`. You can change the remote from "origin" to "myremote" by running `yarn deploy:github --github-remote=myremote`. `yarn deploy:github` uses the github remote to compute the default value for SNOWPACK_PUBLIC_BASE_URL.
+
+Before each of these steps, the script will ask you if you are sure you want to continue. You can make it stop asking by providing `--build` or `--deploy` arguments to `yarn deploy:github`.
```

Comment:
#  Easy Deployments

There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.

<br />

##  Prerequisites

-  [Fork this repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
- Follow the instructions in [docs/build-from-source.md](docs/build-from-source.md)


There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.

<br />

##  Supported Platform

  - [GitHub Pages](#github-pages): A free, easy-to-use [web hosting service](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)

<br />

## Github Pages

###  Technical Limitations

Github pages have same [usage limits](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits).

In general terms, the approximately **20.000 NEW** users can visit your app each month<sup>1</sup>. If more than 20.000 user visit your app each month, you consider other paid hosting solutions.

<sup>1</sup> *Based on 100GB Github Pages bandwidth / 5MB ott-web-app build dir size and assuming perfect browser cache by all visitors.*

<br />

###  Usage Instructions

1. Confirm that you can [build this project from source](./build-from-source.md). You will need to install optional dependencies.
2. Enable the [Github Pages feature](https://docs.github.com/en/pages/getting-started-with-github-pages) for the `gh-pages` branch in your repository.
3. Run `yarn deploy:github`. Be sure to follow the instructions that appear on the screen.
4. (Optional) If you need a customization option, review `yarn deploy:github --help`.

<br />

###  Technical Documentation

The `yarn deploy:github` command executes a simple nodejs script located in `scripts/deploy-github.js`. The script executes the following commands:

1. Runs `yarn build` with `SNOWPACK_PUBLIC_BASE_URL` envvar.<br/><br />That envvar is used to set the URL location of the project. By default if `SNOWPACK_PUBLIC_BASE_URL` is empty, the value will be based on the `git remote get-url origin` command. You can also pass your own `SNOWPACK_PUBLIC_BASE_URL` envvar by running `SNOWPACK_PUBLIC_BASE_URL=/my-base/ yarn deploy:github`.

2. Runs `yarn gh-pages -o origin -d build`.<br /><br />You can change the remote from *origin* to *myremote* by running `yarn deploy:github --github-remote=myremote`.  The `yarn deploy:github` command uses the GitHub remote to compute the default value for `SNOWPACK_PUBLIC_BASE_URL`.

<br />

>**TIP**: Before each of the previous steps, the script will ask if you want to continue. You can prevent these confirmation inquiries by providing `--build` or `--deploy` arguments to `yarn deploy:github`.

<br />

Reply:
Thanks Kevin! That's definitely better. I took your suggestions almost unchanged, I only removed the leading spaces from each line since I think they have no semantic meaning in markdown, and smiliarly changed two spaces after # to a single space, otherwise literally copy pasted it